style.json

```diff
@@ -1,25 +1,24 @@
 {
   "version": 8,
-  "name": "Fluds",
+  "name": "Floods",
   "metadata": {
-    "maputnik:license": "https://github.com/maputnik/osm-liberty/blob/gh-pages/LICENSE.md",
     "maputnik:renderer": "mbgljs",
     "openmaptiles:version": "3.x"
   },
   "sources": {
-    "fludpoints": {
+    "floodpoints": {
       "type": "geojson",
       "data": "https://raw.githack.com/HarelM/IHMMapboxMapExample/main/source.geojson"
     }
   },
-  "sprite": "https://maputnik.github.io/osm-liberty/sprites/osm-liberty",
```

Comment:
Are you sure a style will work with an empty sprite?

Reply:
[`sprite` is optional](https://docs.mapbox.com/mapbox-gl-js/style-spec/root/#sprite) and can safely be removed.
I think IHM overwrites it anyway, isn't it?

Reply:
Right... Forget that...